modules/cnv-configuring-selinux-hpp-on-rhcos8.adoc

```diff
@@ -19,6 +19,10 @@ If you do not use Red Hat Enterprise Linux CoreOS workers, skip this procedure.
 * Create a backing directory on each node for the PersistentVolumes (PVs)
 that the hostpath provisioner creates.
 
+[WARNING]
+====
+If you select a directory that shares space with your Operating System, you can potentially exhaust the space on that partition and your node will become non-functional. It is recommended you create a separate partition and point the hostpath provisioner there so it will not interfere with your Operating System.
```

Comment:
I had a few nits, so I'm proposing a slight rewrite for you to take or leave: `If you select a directory that shares space with your operating system, you can exhaust the space on that partition, causing the node to stop functioning. To avoid this issue, create a separate partition and point the hostpath provisioner to that directory.`